src/execution_plan/execution_plan_modify.c

```diff
@@ -127,8 +112,52 @@ const char **_ExecutionPlan_LocateReferences(OpBase *root, OpBase **op, rax *ref
 	return seen;
 }
 
-void _OpBase_RemoveChild(OpBase *parent, OpBase *child) {
-	_OpBase_RemoveNode(parent, child);
+/* Returns true if the given operation is allowed to have NULL children. */
+static bool _DontMigrateChildren(OpBase *op) {
+	if(op->type == OPType_MERGE) return true;
+	return false;
+}
```

Comment:
Let's discuss this design, I would like to avoid the situation where an operation has a NULL child.